sdk/containerregistry/container-registry/README.md

```diff
@@ -0,0 +1,80 @@
+# Azure Container Registry client library for JavaScript
+
+Azure [Container Registry](https://azure.microsoft.com/services/container-registry/) is a managed, private Docker registry service based on the open-source Docker Registry 2.0. Create and maintain Azure container registries to store and manage your private Docker container images and related artifacts.
+
+Use Azure container registries with your existing container development and deployment pipelines, or use Azure Container Registry Tasks to build container images in Azure. Build on demand, or fully automate builds with triggers such as source code commits and base image updates.
+
+## Getting started
+
+### Currently supported environments
+
+- Node.js version 8.x or higher
+
+### Prerequisites
+
+- An [Azure subscription][azure_sub].
+- An [Azure Container Registry resource][acr_resource]
+
+Usually you'd put a shell command for provisioning the necessary Azure services here.
```

Comment:
Is this a future TODO? :smile:

Reply:
I will need to do more work on README/Samples etc. before a release so just adding a place holder project for now.